examples/ignore-side-effects/main.go

```diff
@@ -0,0 +1,46 @@
+package main
```

Comment:
nit: would prefer a toy shell script synth to avoid an extra image build during the smoke tests

Reply:
I'll track this in #208